packages/fast-usdc/src/cli/operator-commands.js

```diff
@@ -99,6 +99,7 @@ export const addOperatorCommands = (
     .requiredOption('--chainId <string>', 'chain id', Number)
     .requiredOption('--amount <number>', 'number', parseNat)
     .requiredOption('--forwardingAddress <string>', 'bech32 address', String)
+    .requiredOption('--sender <string>', 'Ethereum address initiating', String)
```

Comment:
@samsiegart I wonder with all these options if we should just read the `CctpTxEvidence` from stdin

Reply:
I had it all in one arg at first... But note that plain JSON doesn't suffice. It has bigints in it. I used LegibleCapData... But I suppose bigints occur in just a handful of places and we could convert them from strings or safe integers.

The shape is handy for validating this sort of thing.

Reply:
Could you describe more how that would look usage-wise?

Reply:
> that plain JSON doesn't suffice. … could convert them from strings

Yeah, that's what the new fns in `js-utils` do. It is a different encoding than CapData, but it doesn't really affect consumers unless they need the values to be bigint instead of string. I figure at some point we could have a `coerceJson(CctpTxEvidenceShape)` that figures out which paths should be bigint, etc.

>Could you describe more how that would look usage-wise?

The command would read stdin and the JSON piped to it would be in the shape of `CctpTxEvidence` but with bigints encoded as strings. Their code could use `stringifyWithBigint` or inline that.

Reply:
I think in terms of using a CLI it's still easier to have the args. With JSON we lose the handy docs and parsing/type enforcement, and the overall input length is still pretty much the same.

If we're developing this to be called by another program, should we just expose the stuff below line 122 as a function export? I'm a little unclear on the intended usage pattern.

Reply:
Good points. I agree completely. I'll resolve this therad.